src/cargo/util/toml_mut/dependency.rs

```diff
@@ -279,7 +279,13 @@ impl Dependency {
                     let src = WorkspaceSource::new();
                     src.into()
                 } else {
-                    anyhow::bail!("Unrecognized dependency source for `{key}`");
+                    let mut msg = format!("Unrecognized dependency source for `{key}`");
```

Comment:
Not your mistake but if you are willing to, could you lower the case for the first word? Cargo has a convention starting with lowercase for the first sentence of the message (though not enforced everywhere 😆).

Reply:
Updated!